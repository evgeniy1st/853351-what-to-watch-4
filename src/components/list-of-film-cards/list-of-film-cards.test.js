import React from "react";
import renderer from "react-test-renderer";
import ListOfFilmCards from "./list-of-film-cards.jsx";
import {BrowserRouter} from "react-router-dom";

it(`render ListOfFilmCards`, () => {
  const mockFilms = [
    {
      title: `Fantastic Beasts: The Crimes of Grindelwald`,
      preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
      image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
    },
    {
      title: `Fantastic Beasts: The Crimes of Grindelwald2`,
      preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
      image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
    }
  ];

  const tree = renderer
    .create(<BrowserRouter>
      <ListOfFilmCards
        films={mockFilms}
        onTitleClick={() => { }}
      />
    </BrowserRouter>)
    .toJSON;

  expect(tree).toMatchSnapshot();
});
