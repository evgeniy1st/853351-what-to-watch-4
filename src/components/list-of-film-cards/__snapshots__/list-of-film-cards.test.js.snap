// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ListOfFilmCards 1`] = `[Function]`;
